src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -133,26 +133,47 @@
           </v-row>
         </template>
 
-        <div
-          v-if="minimumThreeDirectorError"
+        <MessageBox
+          v-if="hasAttemptedSubmission && minimumThreeDirectorError"
           id="minimum-three-director-error"
-          class="my-6"
+          class="mt-6"
+          color="red"
         >
```

Comment:
Update the error message format and wording:
![image](https://github.com/user-attachments/assets/7138c679-49cb-4864-bd7b-d46539a24336)